build-release-tools/docker_build.sh

```diff
@@ -22,6 +22,9 @@ IS_OFFICIAL_RELEASE=${IS_OFFICIAL_RELEASE:=false}
 BUILD_NIGHTLY=false
 BUILD_LATEST=false
 
+BUILD_ID=env $BUILD_ID
```

Comment:
Is this trying to get the value of BUILD_ID from the environment?  If I am recalling the env command correctly, I think this is going to try to run the command $BUILD_ID.

```
ohrenj@lab16:~$ env $USER
env: ‘ohrenj’: No such file or directory
```